File: app/src/main/java/ltd/kaizo/mynews/Model/notification/Androidjob/AndroidJobCreator.java
package ltd.kaizo.mynews.Model.notification.Androidjob;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.evernote.android.job.Job;
import com.evernote.android.job.JobCreator;

/**
 * The type Android job creator.
 */
public class AndroidJobCreator implements JobCreator {
    @Nullable
    @Override
    public Job create(@NonNull String tag) {
        return new NytShowNotificationJob();
    }
}

File: app/src/main/java/ltd/kaizo/mynews/controller/ui/notification/AndroidJobCreator.kt
package ltd.kaizo.mynews.controller.ui.notification

import com.evernote.android.job.Job
import com.evernote.android.job.JobCreator

/**
 * The type Android job creator.
 */
class AndroidJobCreator : JobCreator {
    override fun create(tag: String): Job? {
        return NytShowNotificationJob()
    }
}
